### PR TITLE
Adding zones metrics

### PR DESCRIPTION
**Pull request by rohit-dimagi**
* Adding metrics for zone capture.

### CODE REVIEW

File: roles/logstash/templates/90-outputs.conf.j2 [hunk 1]
```diff
@@ -42,10 +42,20 @@ output {
...
+            format => "nginx.error_logs:1|c|#environment:{{ env_monitoring_id }},domain:%{domain},http_method:%{verb},log_level:%{err_severity},error_type:%{err_message},group:proxy"
         }
         id => "logstash-nginx-error_logs"
       }
+
+    udp {
+        host => "{{ datadog_statsd_host|default('localhost') }}"
+        port => {{ datadog_statsd_port|default(8125) }}
+        codec => plain {
+            format => "nginx.request_zone:1|c|#environment:{{ env_monitoring_id }},domain:%{domain},http_method:%{verb},zone:%{zone},group:proxy"
```
Comment (snopoke) [inline roles/logstash/templates/90-outputs.conf.j2]: can we rename this:
```suggestion
            format => "nginx.rate_limiting:1|c|#environment:{{ env_monitoring_id }},domain:%{domain},http_method:%{verb},zone:%{zone},group:proxy"
```

**Review by snopoke — APPROVED** [review]
small change

**Review by snopoke — COMMENTED** [review]
Just thinking that this metric should only be send if the log line was from rate limiting. Currently it will be sent for every error log line.

Maybe wrap the output in an if statement?
```
if [zone] {
...
}
```

**Comment by rohit-dimagi** [comment]
> Just thinking that this metric should only be send if the log line was from rate limiting. Currently it will be sent for every error log line.
> 
> Maybe wrap the output in an if statement?
> 
> ```
> if [zone] {
> ...
> }
> ```

Yeah you are right. added it in zone if block

**Pull request merged**